firecrown/likelihood/gauss_family/statistic/README.md

```diff
@@ -0,0 +1,73 @@
+```
+s_count.add_tracer('misc', 'cluster_counts_true_mass', metadata=metadata).
+```
+
+Having the tracer, the user must call the `add_data` function for all the data points using the same tracer. The data should be the number of clusters in each bin of redsfhit and proxy. The order of the data is really important: the user
+must add the data for each redshift bin iterating then over the proxy bins, that is, filling a row-major (z, proxy) matrix.
+
+The function to call the data must be such as
+```
+cluster_count = sacc.standard_types.cluster_mass_count_wl
+for i in range(z_bins):
+    for i in range(proxy_bins):
+        tracer = 'cluster_counts_true_mass'
+        value = data_lz[i][j]
+        s_count.add_data_point(cluster_count, (tracer,), value, err=1.)
+```
+In the above, the `cluster_count` variable represents the type of data point. The SACC library already has some `standard_types` and thus the data should be added with the `cluster_mass_count_wl`
+option. For the supported types in Firecrown, check [SupportedDataTypes](cluster_number_counts_enum.py). The `value` should be the number counts data in each bin of redsfhit and proxy.
+
+ Lastly, the user must add the covariance as a `numpy.ndarray` and call the following functions to save the file:
+ ```
+ s_count.add_covariance(covariance)
+ s_count.to_canonical_order()
+ s_count.save_fits("clusters.sacc", overwrite=True)
+ ```
+
+# Documentation for clusters_number_counts.py
+
+This module reads the necessary data from a SACC file to compute the theoretical prediction of cluster number counts inside bins of redshift and a mass proxy. The `NumberCountStat` class must be initialized with:
+
+*`sacc_tracer : str`. The user must provide the tracer's name. Following the documentation above it should be a `cluster_counts_true_mass` string.
+*`sacc_data_type : str`. The type of SACC data. So far, following the documentation above, this entry should be a string  `cluster_mass_count_wl`.
+*`number_density_func`: CCLDensity object. The number density function used
+to compute the theoretical prediction of the cluster number counts. These functions can be found in firecrown/models/ccl_density.py.                      
+*`systematics: Optional[List[Systematic]]`. This entry is optional. This option represents the implementation of a shift or a bias in the redshift or the proxy. No method utilizes this option so far in this class.
+
+## Read Function
+The read function takes a SACC file and extracts the necessary data to perform a cluster number counts analysis. With some pre-defined functions from the SACC library, we extract:
+
+*The `metadata` from the `sacc_tracer` given by the user. It should contain the redshift and proxy bins and types, and also the sky area.
+Check the above documentation for further information on the `metadata`.
+*The `nz`, that is, the number of clusters in each bin, which is stored in the `data.value` in the SACC file.
```

Comment:
To me, `nz` sounds more like n(z) than like a count in a z-proxy bin. Maybe we should think of a more explicit variable name.

Reply:
I feel that a variable that represents a number should relate somehow to `n`. So `n_cl` could be number of clusters,  `n_zp`, that could be in bin of redshift and the proxy, or `n_bin`, which is more general.  What do you think? @vitenti @marcpaterno @combet

Reply:
When I see `n` I think density, not number. `N` feels more like a number to me, but I'm not sure of the FireCrown rules regarding capitalization. Also, for a lot of poeple `zp` might mean photometric redshift...

Reply:
Why not use `count`?